test/integration/test_integration.py

```diff
@@ -32,25 +32,56 @@ def parse_results(log):
     return yaml.load('\n'.join(results))['results']
 
 
-def run_task(nevr):
+def run_task(nevr, *, reterr=False):
     '''
     Run the task on a Koji build.
     Returns a dict with Results (outcome, artifact, item)
+    If reterr is true, returns a tuple with the above and captured stderr
+    If reterr is false, prints the stderr
     '''
     proc = subprocess.Popen(
         ['runtask', '-i', nevr, '-t', 'koji_build', 'runtask.yml'],
         stderr=subprocess.PIPE,
         universal_newlines=True,
     )
     _, err = proc.communicate()
-    print(err, file=sys.stderr)
     if proc.returncode != 0:
         raise RuntimeError('runtask exited with {}'.format(proc.returncode))
     results = parse_results(err)
 
-    return {r['checkname']: Result(r.get('outcome'),
-                                   r.get('artifact'),
-                                   r.get('item')) for r in results}
+    ret = {r['checkname']: Result(r.get('outcome'),
+                                  r.get('artifact'),
+                                  r.get('item')) for r in results}
+
+    if reterr:
+        return ret, err
+
+    print(err, file=sys.stderr)
+    return ret
+
+
+@pytest.fixture()
+def tracer_results(_tracer_results):
+    '''This should FAIL the two_three check'''
+    print(_tracer_results[1], file=sys.stderr)
+    return _tracer_results[0]
```

Comment:
@irushchyshyn This fixture will be executed on every test that uses it. Is uses the `_tracer_results` fixture, hence it will always have the result of that available. It will print the captured stderr and return the dict with results.